### ISSUE TITLE
CSRF error when overloading SonataUser's registration form

### ISSUE THREAD

**Issue by ima-tech** [sonata-project/SonataUserBundle#392]
Before you get excited , I have added form_rest in my page !
____
I have created a github repository. If you pull it, you will see the errors for yourself: https://github.com/ima-tech/testSonataUser

Also, I have also asked this question on StackOverflow. I don't know the policy about posting URL from StackOverflow here, I just feel that it is easier to read on StackOverflow(http://stackoverflow.com/questions/22950553/symfony2-overload-registration-form-causes-csrf-errors)

Basically, I have overwritten the registration form so that I can add multiple fields such as country, province, postal code, etc...

The form is showing with all the fields I added. However, when I click on register, I get the error saying: "Le jeton CSRF est invalide. Veuillez renvoyer le formulaire."

I really don't know what to do next...
